geopyspark/geotrellis/rdd.py

```diff
@@ -401,6 +401,25 @@ def reproject(self, target_crs, extent=None, layout=None, scheme=FLOAT, tile_siz
 
         return TiledRasterRDD(self.geopysc, self.rdd_type, srdd)
 
+    def lookup(self, col, row):
+        """Return the value(s) in the image of a particular SpatialKey (given by col and row)
+
+        Args:
+            col: The SpatialKey column
+            row: The SpatialKey row
+
+        Returns: An array of numpy arrays (the tiles)
+        """
+        if self.rdd_type != SPATIAL:
+            raise ValueError("Only TiledRasterRDDs with a rdd_type of Spatial can use lookup()")
+
```

Comment:
We can get the `Bounds` from the `layer_metadata`, so it might be good to check to see if the `col` and `row` given fall within the `Bounds`. This would return a more readable and understandable stack trace if an Exception is raised.